include/uapi/sound/sof/tokens.h

```diff
@@ -24,6 +24,9 @@
 #define SOF_TPLG_KCTL_ENUM_ID	257
 #define SOF_TPLG_KCTL_BYTES_ID	258
 #define SOF_TPLG_KCTL_SWITCH_ID	259
+#define SOF_TPLG_KCTL_BYTES_VOLATILE_RO 260
```

Comment:
@RDharageswari the last sentence in the commit message is not necessary. How it is used depends on the user really

Reply:
done